README.md

```diff
@@ -150,6 +151,40 @@ tilesets delete-source
 
 Permanently delete a tileset source and all of its files. This is not a recoverable action!
 
+### estimate-area 
+
+```shell
+tilesets estimate-area <features> -p/--precision <precision>
+```
+
+Estimates area (km<sup>2</sup>) of features with a specified precision. Accepts line-delimited GeoJSON or GeoJSON feature collections as files or via `stdin` or a list of string-encoded coordinate pairs of the form "[lng, lat]", or "lng, lat", or "lng lat". Precision must be 10m, 1m, 30cm or 1cm.
+
+Flags:
```

Comment:
Should the `--precision` flag be documented in this list as well? I found it hard to find the possible options in the description above.

Additionally, it'd be nice to have a `[required]` note next to it if it is required, or note if there's a default (similar to the [create](https://github.com/mapbox/tilesets-cli#create) command)